css/main.css

```diff
@@ -6,6 +6,10 @@
   box-sizing: border-box;
```

Comment:
- [ ] There is an issue with the background for the intro section. It overlays the header and the links are almost unnoticeable, this could bring user experience problems. I would recommend removing this `svg` image from the HTML and using it in the CSS with the `background` property.

![image](https://user-images.githubusercontent.com/81522435/135772248-3d8bbc8d-2cca-4c1d-bd60-f6ac5904ed19.png)